CMakeLists.txt

```diff
@@ -526,38 +526,38 @@ set_target_properties(maskpass PROPERTIES
 install(TARGETS maskpass DESTINATION ${YDB_INSTALL_DIR}/plugin/gtmcrypt)
 
 # Always copy files into the plugin directory
-foreach(f
-      Makefile.mk
-      encrypt_sign_db_key.sh
-      gen_keypair.sh
-      gen_sym_hash.sh
-      gen_sym_key.sh
-      gtm_tls_impl.c
-      gtm_tls_impl.h
-      gtm_tls_interface.h
-      gtmcrypt_dbk_ref.c
-      gtmcrypt_dbk_ref.h
-      gtmcrypt_interface.h
-      gtmcrypt_pk_ref.c
-      gtmcrypt_pk_ref.h
-      gtmcrypt_ref.c
-      gtmcrypt_ref.h
-      gtmcrypt_sym_ref.c
-      gtmcrypt_sym_ref.h
-      gtmcrypt_util.c
-      gtmcrypt_util.h
-      import_and_sign_key.sh
-      maskpass.c
-      minimal_gbldefs.c
-      pinentry-gtm.sh
-      pinentry.m
-      show_install_config.sh
-      ydb_getenv.c
-      ydb_getenv.h
-      ydb_logicals.h
-      ydb_logicals_tab.h
+foreach(f_in
+      ${YDB_SOURCE_DIR}/sr_unix/Makefile.mk
```

Comment:
I'll add something like this:

The cmake copy command does not support wildcards (https://cmake.org/cmake/help/v3.12/manual/cmake.1.html, and https://gitlab.kitware.com/cmake/cmake/merge_requests/687/diffs); in some cases, wildcards may work because the generator calls a shell which expands the wildcard, but this is not always true. In the case of NInja, the wildcard is evidently not expanded by the shell.

Reply:
Thanks. That works.